bundles/org.eclipse.packagedrone.utils.rpm/src/org/eclipse/packagedrone/utils/rpm/build/DigestAlgorithm.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2016 IBH SYSTEMS GmbH and others.
```

Comment:
For a new file this should be "Red Hat Inc and others"

Reply:
Fixed. I've also updated the year to 2018.